SUMMARY.md

```diff
@@ -1,6 +1,10 @@
 # Table of contents
 
-* [Welcome to Mayastor!](README.md)
+* [Welcome to Mayastor!](README.md)]
```

Comment:
There is an extra ']' at the end.

Reply:
Good spot, thank you.  I've corrected it.